rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2022-08-18"
+channel = "nightly-2022-08-19"
```

Comment:
Haha 🚀

Reply:
Upsi

Reply:
Yeah, somehow 08-18 doens't work for me. Never found out why. Fixed, sorry :P